### PR TITLE
A way to forget coffin poles

### PR DESCRIPTION
**Pull request by josephwright**
https://github.com/latex3/latex3/issues/442

### CODE REVIEW

**Review by FrankMittelbach — COMMENTED** [review]
maybe I'm blind but I only see changes "using" the new function but nowhere its definition. Am I missing somehting obvious?

**Comment by josephwright** [comment]
> maybe I'm blind but I only see changes "using" the new function but nowhere its definition. Am I missing somehting obvious?

It's a rename: current internal `\@@_(g)update:N` to public `\coffin_(g)reset_poles:N`

**Comment by FrankMittelbach** [comment]
> It's a rename: current internal `\@@_(g)update:N` to public `\coffin_(g)reset_poles:N`

so obviously blind ... as I suspected

File: l3kernel/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ this project uses date-based 'snapshot' version identifiers.
 ### Added
 - Rounded out the `\use_i:nn...` functions to cover all cases taking
   one argument from up to nine arguments
+- `\coffin_(g)reset_poles:N`
```
Comment (FrankMittelbach) [inline l3kernel/CHANGELOG.md:13]: might state that the old name was

**Review by FrankMittelbach — APPROVED** [review]

File: l3kernel/CHANGELOG.md (hunk 1 above)
Comment (josephwright) [reply l3kernel/CHANGELOG.md:13]: It was internal so I'm not sure that's relevant

**Comment by josephwright** [comment]
Everyone happy with the proposed name here?

**Comment by FrankMittelbach** [comment]
> Everyone happy with the proposed name here?

I am

**Pull request merged**